src/lemlib/movements/purepursuit.cpp

```diff
@@ -101,22 +102,22 @@ std::pair<int, int> PurePursuit::update(Pose pose) {
         Pose intersect = circleLineIntersect(last, cur, pose, lookaheadDist);
         // if the intersection is not the robot's current position, then we have found the lookahead point
         if (intersect != pose) {
-            lookahead = {intersect.x, intersect.y, 0, 0, i};
+            lookahead = {intersect.x, intersect.y, 0_deg, 0_rpm, i};
             break;
         }
     }
     prevLookahead = lookahead; // update previous lookahead position
 
     // get the curvature of the arc between the robot and the lookahead point
-    float curvature = getCurvature(pose, lookahead);
+    Curvature curvature = getCurvature(pose, lookahead);
 
     // get the target velocity of the robot
-    float targetVel = closest.speed;
+    AngularVelocity targetVel = closest.speed;
```

Comment:
```suggestion
    LinearVelocity targetVel = closest.speed;
```
Change to correct unit